tools/pd-ut/ut.go

```diff
@@ -0,0 +1,801 @@
+// Copyright 2024 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"encoding/xml"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path"
+	"regexp"
+	"runtime"
+	"strings"
+	"sync"
+	"time"
+
+	// Set the correct value when it runs inside docker.
+	_ "go.uber.org/automaxprocs"
+)
+
+func usage() bool {
+	msg := `// run all tests
+pd-ut
```

Comment:
I'd prefer to name this as `dev`, as we can do more  scope with `dev` command , such as

```
./dev lint
./dev test pkg/util
./dev generator swagger
./dev bench heartbeat
```

Reply:
maybe can unify all tools in another pr